frontend/screens/AlbumScreen.tsx

```diff
@@ -230,7 +230,7 @@ const AlbumScreen = ({ route, navigation }) => {
       return (
         <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
           <Text style={{ fontSize: 18 }}>
-            Access token outdated!
+            {data.message}
```

Comment:
Not alwas an access token error, could be ML server can't be reached from Node, etc.

Reply:
💀